public/pages/CreateTrigger/containers/ConfigureActions/ConfigureActions.js

```diff
@@ -159,34 +193,14 @@ class ConfigureActions extends React.Component {
   };
 
   loadDestinations = async (searchText = '') => {
-    const { httpClient, values, arrayHelpers, notifications, fieldPath, flyoutMode } = this.props;
+    const { values, arrayHelpers, notifications, fieldPath, flyoutMode } = this.props;
     const { allowList, actionDeleted } = this.state;
 
     this.setState({ loadingDestinations: true });
     try {
-      const response = await httpClient.get('../api/alerting/destinations', {
```

Comment:
Could you clarify why this was removed? We didn't deprecate the alerting plugin destinations in 3.0 to my knowledge.